contracts/test/integration/_block.txt

```diff
@@ -0,0 +1 @@
+14207980
```

Comment:
That would necessarily require hardcoding the env variable as a magic number in a few places - CI, package.json - and hide it away almost as a hard coded global. I want to make it obvious where it is and have one place to change it

Reply:
It's an anti-pattern really to have hardcoded `env` variables around in scripts

Reply:
I'd like the default to match the default of the hardhat e2e tests - that is, they fork from the most recent mainnet block and fail-fast so we know when things break. (Then, if a hardcoded block *is* set, use that instead)

Reply:
I like having it in a file rather than as an env variable, makes it obvious & shows in diff etc.

That being said, I didn't think about the possibility of *not* providing a block & forking for most recent mainnet block. That's a good idea

Reply:
Removed as discussed